src/index.js

```diff
@@ -0,0 +1,324 @@
+const path = require('path');
+const inquirer = require('inquirer');
+
+const Manager = require('./lib/Manager');
+const Engineer = require('./lib/Engineer');
+
+const Intern = require('./lib/Intern');
+const fs = require('fs');
+const generateHTML = (manager, engineers, interns) => {
+  const generateEngineerCard = engineers
+    .map((engineer) => {
+      return `<div class="card" style="width: 18rem">
+  <div class="card-body">
+    <h5 class="card-title">engineer</h5>
+
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">${engineer.id}</li>
+      <li class="list-group-item">email:${engineer.email}</li>
+    </ul>
+  </div>
+</div>`;
+    })
+    .join('');
+
+  const generateInternCard = interns
+    .map((intern) => {
+      `<div class="card" style="width: 18rem">
+  <div class="card-body">
+    <h3 class="card-title">${intern.name}</h3>
+    <h5 class="card-title">intern</h5>
+
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">${intern.id}</li>
+      <li class="list-group-item">email: ${intern.email}</li>
```

Comment:
`mailto:` should be used for emails so the user can click on the link and be taken directly to their emails.